src/nvim/os/stdpaths.c

```diff
@@ -108,6 +108,34 @@ char *get_xdg_home(const XDGVarType idx)
   return dir;
 }
 
+/// Return nvim log directory
+///
+/// @return [allocated] "$XDG_CACHE_HOME/nvim/logs"
+char *get_nvim_logpath(void)
+  FUNC_ATTR_WARN_UNUSED_RESULT
+{
+  char *dir = stdpaths_get_xdg_var(kXDGCacheHome);
+  if (dir) {
+    dir = concat_fnames_realloc(dir, "nvim", true);
+    dir = concat_fnames_realloc(dir, "logs", true);
+  }
+  else {
+    dir = "";
+  }
+  return dir;
```

Comment:
I feel like in retrospect this is dumb and I should have just hardcoded this. Are there any exceptions to the pathing other than windows? I'm not super familiar with the bsds.